index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+< lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>NINA SAALFRANK</title>
+  </head>
+
+
+  <!-- HERE WE GO-->
+
+  <body>
+    <header>
+      <h1>NINA SAALFRANK</h1>
+
+      <!--NAVIGATION-->
+
+      <nav id="nav">
+        <a href="#about">ABOUT</a>
+        <a href="#web">WEB</a>
+        <a href="#design">DESIGN</a>
+        <a href="#photo">PHOTO</a>
+        <a href="#contact">CONTACT</a>
+      </nav>
+    </header>
+
+    <!--ONE PAGER START-->
+
+    <main>
+      <!--ABOUT-->
+
+      <section id="about">
+        <h2>ABOUT</h2>
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolorem,
+          quod ipsa corporis voluptas temporibus perferendis nobis magnam nam
+          quibusdam animi. Ipsa ullam praesentium veritatis dignissimos commodi
+          dicta ut eaque laboriosam.
+        </p>
+      </section>
+
+      <!--WEB-->
+      <section id="web">
+        <h2>WEB</h2>
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolorem,
+          quod ipsa corporis voluptas temporibus perferendis nobis magnam nam
+          quibusdam animi. Ipsa ullam praesentium veritatis dignissimos commodi
+          dicta ut eaque laboriosam.
+        </p>
+        <a href="https://www.google.com">&#187; see more</a>
+      </section>
+
+      <!--DESIGN-->
+      <section id="design">
+        <h2>DESIGN</h2>
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolorem,
+          quod ipsa corporis voluptas temporibus perferendis nobis magnam nam
+          quibusdam animi. Ipsa ullam praesentium veritatis dignissimos commodi
+          dicta ut eaque laboriosam.
+        </p>
+        <a href="https://www.google.com">&#187; see more</a>
+      </section>
+
+      <!--PHOTO-->
+      <section id="photo">
+        <h2>PHOTO</h2>
+
+        <img
+          src="https://images.pexels.com/photos/15394525/pexels-photo-15394525.jpeg?auto=compress&cs=tinysrgb&w=800&lazy=load"
+          alt="photo1"
+          height="200px"
+        />
+
+        <img
+          src="https://images.pexels.com/photos/15394524/pexels-photo-15394524.jpeg?auto=compress&cs=tinysrgb&w=800&lazy=load"
+          alt="photo2"
+          height="200px"
+        />
+
+        <img
+          src="https://images.pexels.com/photos/15394523/pexels-photo-15394523.jpeg?auto=compress&cs=tinysrgb&w=800&lazy=load"
+          alt="photo2"
+          height="200px"
+        />
+
+        <p><a href="https://www.google.com">&#187; see more</a></p>
```

Comment:
I think the "see more" should go between <a> and </a> or is this something we haven't learned so far?